### ISSUE TITLE
Question: does the inverse of this extension exist?

### ISSUE THREAD

**Issue by joeskeen** [microsoft/vscode-vs-keybindings#23]
I know this is likely not the right place for this question, but I'm not sure where the right place would be.

From my understanding this extension brings Visual Studio keybindings to Visual Studio Code.  I would like to do the opposite: I'm familiar with VSCode keybindings and would like to use them in Visual Studio (which I very rarely open anymore).  Is anyone aware of a plugin/extension/way of accomplishing that?  If not, what would it take to make something like that?

Thanks!

**Comment by connorads** [comment]
Yes @joeskeen I believe this is already build into Visual Studio 2019 (tested on 16.4.3 Professional).

`Tools > Options > Environment > Keyboard`
![image](https://user-images.githubusercontent.com/10026538/72988008-63d51180-3de3-11ea-866c-c125a4ae2b2e.png)

**Comment by aberus** [comment]
Ia agree with @greenfox1505 in https://github.com/microsoft/vscode-vs-keybindings/issues/28#issuecomment-626935124: "That should be linked in the `README.md`." as this issues was mentioned mulitple times. 

@connorads what do you think?
